src/activities/Elsa.Activities.Http/Formatters/XMLContentFormatter.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Serialization;
+using Elsa.Activities.Http.Services;
+
+
+namespace Elsa.Activities.Http.Formatters
+{
+    public static class XMLContentFormatterCache
+    {
+        public static List<Type> TypeCache { get; set; }
+    }
+
+
+    public class XMLContentFormatter : IContentFormatter
```

Comment:
Please update your fork with latest commits from `develop`, then implement `IHttpRequestBodyParser`.
Sorry for the inconvenience.

Reply:
Yup, will do so....